indico/MaKaC/webinterface/tpls/events/include/Common.tpl

```diff
@@ -1,9 +1,5 @@
 <%def name="renderDescription(text)">
-    % if isStringHTML(text):
-        <span itemprop="description">${text}</span>
-    % else:
-        <pre itemprop="description">${text}</pre>
-    % endif
+    <span itemprop="description">${text}</span>
```

Comment:
Are you sure you want to include this? It's the commit that broke linebreaks in descriptions.